renderdoc/driver/gl/gl_driver.cpp

```diff
@@ -2562,13 +2567,11 @@ void WrappedOpenGL::CreateVRAPITextureSwapChain(GLuint tex, GLenum textureType,
 
   if(textureType == eGL_TEXTURE_2D_ARRAY)
   {
-    Common_glTextureImage3DEXT(id, eGL_TEXTURE_2D, 0, internalformat, width, height, 2, 0, eGL_RGBA,
-                               eGL_UNSIGNED_BYTE, NULL);
+    Common_glTextureStorage3DEXT(id, eGL_TEXTURE_2D_ARRAY, levels, internalformat, width, height, 2);
```

Comment:
Is it valid to change to use `ARB_texture_storage` here - can we guarantee that the extension will be available on all platforms using vrapi? Is there a reason to require it, over using plain `glTexImage` ?